backend/tracim_backend/lib/core/content.py

```diff
@@ -797,6 +795,7 @@ def get_pdf_preview_path(
         :return: preview_path as string
         """
         file_path = self.get_one_revision_filepath(revision_id)
+        page = preview_manager_page_format(page)
```

Comment:
`page` is probabbly `page_id` or`page_number`? Do not use page as it is intuitively an object (and here, my feeling is that it is a number)